llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -2379,6 +2381,11 @@ bool VPlanTransforms::tryAddExplicitVectorLength(
       CanonicalIVIncrement->getDebugLoc(), "index.evl.next");
   EVLPhi->addOperand(NextEVLIV);
 
+  VPValue *NextAVL = Builder.createOverflowingOp(
+      Instruction::Sub, {AVLPhi, OpVPEVL}, {/*hasNUW=*/true, /*hasNSW=*/false},
+      DebugLoc(), "avl.next");
```

Comment:
```suggestion
      DebugLoc::getUnknown(), "avl.next");
```

Reply:
TIL about DebugLoc::getUnknown(). I don't see it used anywhere else in the loop vectorizer though, do we need to start using it here?

Reply:
Thanks, I didn't realise it was a recent change. I guess we should start using DebugLoc::getCompilerGenerated() in this instance then?

Reply:
Yes, I think that would be better.